js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetGraphExplorer.tsx

```diff
@@ -700,20 +705,40 @@ const AssetGraphExplorerWithData = ({
                       />
                     </Tooltip>
                   )}
-                  <GraphQueryInputFlexWrap>
-                    <AssetSelectionInput
-                      assets={graphQueryItems}
-                      value={explorerPath.opsQuery}
-                      onChange={onChangeAssetSelection}
-                      onErrorStateChange={(errors: SyntaxError[]) => {
-                        if (errors !== errorState) {
-                          setErrorState(errors);
-                        }
-                      }}
-                    />
-                  </GraphQueryInputFlexWrap>
-                  <CreateCatalogViewButton />
-                  <AssetLiveDataRefreshButton />
+                  {viewType === AssetGraphViewType.CATALOG ? (
+                    <>
+                      {toggleFullScreen ? (
+                        <Tooltip content={isFullScreen ? 'Collapse' : 'Expand'}>
+                          <Button
+                            icon={
+                              <Icon
+                                name={isFullScreen ? 'collapse_fullscreen' : 'expand_fullscreen'}
+                              />
+                            }
+                            onClick={toggleFullScreen}
+                          />
+                        </Tooltip>
```

Comment:
Could see wrapping this into a shared button so that we're consistent about adding the tooltip